### PR TITLE
V0.3.0 release

### PR DESCRIPTION
**Pull request by hendrikebbers**

### CODE REVIEW

File: jvm-manager.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## JVM Manager
+
+<span class="text-highlight">Open<span>WebStart</span></span> from Karakun will provide an integrated JVM Manager.
+The JVM Manager downloads Java versions from a dedicated server, and manages versions internally. 
+With <span class="text-highlight">Open<span>WebStart</span></span>, it will be easy for developers to specify the Java version and vendor to run JNLP-based applications.
+
+The JVM Manager is configured using the  <span class="text-highlight">Open<span>WebStart</span></span> Settings client.
+Within the JVM Manager Settings panel a list of all known JVMs is displayed.  
+The list item (sandwich) menu allows to activate/deactivate and remove a JVM instance from the list.
```
Comment (hendrikebbers) [inline jvm-manager.md]:
```suggestion
The list item (sandwich) menu allows to activate/deactivate and remove a managed JVM instance from the list.
```

File: jvm-manager.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+### Local JVMs
+
+With "Find local" <span class="text-highlight">Open<span>WebStart</span></span> allows to find existing Java installations on your local machine.
```
Comment (hendrikebbers) [inline jvm-manager.md]:
```suggestion
With the "Find local" action <span class="text-highlight">Open<span>WebStart</span></span> allows to find existing Java installations on your local machine.
```

File: jvm-manager.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+All found JVMs are listed and are deactivated by default.
+Removing locally installed JVMs from the list does not delete it from the file system.
+
+"Add local" allows to manually add additional local JVM installations to the list.
```
Comment (hendrikebbers) [inline jvm-manager.md]:
```suggestion
The "Add local" action allows to manually add additional local JVM installations to the list. 
```

File: jvm-manager.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+## JVM Manager
+
+<span class="text-highlight">Open<span>WebStart</span></span> from Karakun will provide an integrated JVM Manager.
+The JVM Manager downloads Java versions from a dedicated server, and manages versions internally. 
+With <span class="text-highlight">Open<span>WebStart</span></span>, it will be easy for developers to specify the Java version and vendor to run JNLP-based applications.
+
+The JVM Manager is configured using the  <span class="text-highlight">Open<span>WebStart</span></span> Settings client.
+Within the JVM Manager Settings panel a list of all known JVMs is displayed.  
+The list item (sandwich) menu allows to activate/deactivate and remove a JVM instance from the list.
+
```
Comment (hendrikebbers) [inline jvm-manager.md]: We should add an image of the JVM Manger panel here.

File: jvm-manager.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+The update strategy for remote JVMs can be one of the following:
+* Ask for update on local match (default)
+* Do nothing on local match
+* Automatically download
+
+To narrow down the JVM download and usage of a specific vendor you can select a vendor from the list of provided vendors.
+If you don't have any specific vendor preference and want to allow <span class="text-highlight">Open<span>WebStart</span></span> to pick a JVM from any vendor select the asterisk.
+The default vendor is set to "AdoptOpenJDK".
+
+The supported JVM version can be specified by providing a valid version string according to JSR-56, Appendix A.
```
Comment (hendrikebbers) [inline jvm-manager.md]: we should add a link to the JSR

Comment (hendrikebbers) [reply jvm-manager.md]: https://www.jcp.org/en/jsr/detail?id=56

Comment (hendrikebbers) [reply jvm-manager.md]: We are not allowed to host the JSR on our own based on Oracles licence:

Oracle America, Inc. hereby grants you a fully-paid, non-exclusive, non-transferable, worldwide, limited license (without the right to sublicense), under Oracle's applicable intellectual property rights to view, download, use and reproduce the Specification only for the purpose of internal evaluation. This includes (i) developing applications intended to run on an implementation of the Specification, provided that such applications do not themselves implement any portion(s) of the Specification, and (ii) excerpting brief portions of the Specification in oral or written communications which discuss the Specification provided that such excerpts do not in the aggregate constitute a significant portion of the Technology. No license of any kind is granted hereunder for any other purpose including, for example, creating and distributing implementations of the Specification, modifying the Specification (other than to the extent of your fair use rights), or distributing the Specification to third parties. Also, no right, title, or interest in or to any trademarks, service marks, or trade names of Oracle or Oracle's licensors, is granted hereunder. If you wish to create and distribute an implementation of the Specification, a license for that purpose is available at http://www.jcp.org. The foregoing license is expressly conditioned on your acting within its scope, and will terminate immediately without notice from Oracle if you breach the Agreement or act outside the scope of the licenses granted above. Java, and Java-related logos, marks and names are trademarks or registered trademarks of Oracle America, Inc. in the U.S. and other countries.

File: jvm-manager.md (hunk 3 above)
Comment (AndreasEhret) [reply jvm-manager.md]: done

File: jvm-manager.md (hunk 2 above)
Comment (AndreasEhret) [reply jvm-manager.md]: done

File: jvm-manager.md (hunk 1 above)
Comment (AndreasEhret) [reply jvm-manager.md]: done

File: jvm-manager.md (hunk 4 above)
Comment (AndreasEhret) [reply jvm-manager.md]: done

File: jvm-manager.md (hunk 5 above)
Comment (AndreasEhret) [reply jvm-manager.md]: As they obviously not provide a navigable online JSR version that we can easily link to and also not want that anybody else host a navigable online JSR version according to Hendriks findings above, I just make a link to the JSR page (https://www.jcp.org/en/jsr/detail?id=56) here

**Pull request merged**